tests/testthat/test-format.R

```diff
@@ -18,22 +18,22 @@ test_that("p_significance", {
   )
   expect_equal(
     format(map_estimate(x)),
-    data.frame(x = "0.13", stringsAsFactors = FALSE),
+    data.frame(MAP_Estimate = "0.13", stringsAsFactors = FALSE),
     ignore_attr = TRUE
   )
   expect_equal(
     format(p_direction(x)),
-    data.frame(x = "51.00%", stringsAsFactors = FALSE),
+    data.frame(Parameter = "Posterior", pd = "51.00%", stringsAsFactors = FALSE),
     ignore_attr = TRUE
   )
   expect_equal(
     format(p_map(x)),
-    data.frame(x = "0.973", stringsAsFactors = FALSE),
+    data.frame(Parameter = "Posterior", p_MAP = "0.973", stringsAsFactors = FALSE),
     ignore_attr = TRUE
   )
   expect_equal(
     format(p_significance(x)),
-    data.frame(x = "0.46", stringsAsFactors = FALSE),
+    data.frame(Parameter = "Posterior", ps = "0.46", stringsAsFactors = FALSE),
```

Comment:
@mattansb @IndrajeetPatil @DominiqueMakowski This is how the package behaviour changed due to #603 - please check if this is intentional.

Furthermore, examples have to be revised, see `?p_map`:

`p_map(x, method = "kernel")` -> `as.numeric(p_map(x, method = "kernel"))` in order to work, since numeric inputs now always return a data frame.

Reply:
yes that's correct